data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -71,6 +71,8 @@ spec:
                       properties:
                         address:
                           type: string
+                        disableCertificateVerification:
```

Comment:
Can we just call this certificateVerification with values "Enabled" and "Disabled" with "Enabled" being the default?

Reply:
I think it's stated in another comment, but Michael went down the bool direction because we were following the pattern set by the baremetal team for the BMC block.
https://github.com/openshift/installer/blob/08aa270e20db77b237306b3f3700d6b2ad654abc/pkg/types/baremetal/platform.go#L15

That said, I don't mind diverging from them. My argument is that we plan on introducing other ways to specify BMC credentials down the line, so I wanted us to have a separate set of supported options that we can maintain as valid, without having to tightly coordinate with them to enable everything we allow for fencing to also work for BMO.

Reply:
@jaypoulz  These are 2 different locations. install-config.yaml is user facing and the type BMC struct {} is an internal data structure. My comment is on what is proposed in install-config.yaml which is used to customize install parameters.